sdk/identity/azure-identity/README.md

```diff
@@ -207,6 +196,40 @@ default_credential = DefaultAzureCredential()
 client = SecretClient("https://my-vault.vault.azure.net", default_credential)
 ```
 
+## Managed Identity Support
+
+The [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) is supported via either the `DefaultAzureCredential` or the `ManagedIdentityCredential` directly for the following Azure Services:
```

Comment:
While you're here, please replace all occurrences of "docs.microsoft.com" with "learn.microsoft.com".